src/isIsogram.test.js

```diff
@@ -6,4 +6,20 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
+  it('should return true if all letters are different', () => {
+    expect(isIsogram('clok')).toEqual(true);
+  });
+
+  it('should return true if string is confused', () => {
+    expect(isIsogram('')).toEqual(true);
+  });
+
+  it('should return false if all letters are repeated', () => {
```

Comment:
```suggestion
  it('should return false if all letters are repeated', () => {
```
in this example not all letters are repeated